src/job/mod.rs

```diff
@@ -439,7 +439,7 @@ pub async fn spawn_process_all_payout_queues(
 }
 
 #[instrument(name = "job.schedule_spawn_process_payout_queue", skip_all, fields(error, error.level, error.message), err)]
-async fn spawn_schedule_process_payout_queue(
+pub async fn spawn_schedule_process_payout_queue(
```

Comment:
This is the wrong spawn. It needs to be spawned onto the account channel straight away. Incase we are triggering a non-manual queue manually (which is allowed) and there is another trigger pending it would wait (since this channel is ordered). So we need to bypass this `scheduling` channel and go straight to the channel where it's actually running.
For more context what the job does that gets scheduled here - it schedules onto another channel for immediate execution.